src/test/java/org/folio/rest/impl/TransformationFieldsServiceTest.java

```diff
@@ -52,7 +52,7 @@ private void validateCountForEachRecordType(TransformationFieldCollection transf
         .equals(TransformationField.RecordType.HOLDINGS))
       .count());
 
-    assertEquals(48, transformationFieldCollection.getTransformationFields()
+    assertEquals(52, transformationFieldCollection.getTransformationFields()
```

Comment:
I missed this during initial review but , holdings count is not modified, that means holding Note types are not generateD?